capirca/lib/fortigate.py

```diff
@@ -677,8 +677,12 @@ def __str__(self):
       self._term.comment += [f"Owner: {self._term.owner}"]
     if self._term.comment and self._term.verbose:
        lines += [f'{_SP * 2} set comments "{self._obj_container.fix_comment_length((" ").join(self._term.comment))}"']
-    lines += [f"{_SP * 2} set srcintf {self._term.source_interface or 'any'}"]
-    lines += [f"{_SP * 2} set dstintf {self._term.destination_interface or 'any'}"]
+    # fortigate local-in policy exception
```

Comment:
I think we need this behavior to be gated inside a platform check for fortigatelocalin 

Like in https://github.com/google/capirca/blob/master/capirca/lib/cisco.py#L660

So that this behavior is only for the local-in policy, and not for the general Fortigate generator (which should have the same behavior as before).